aws/resource_aws_eks_cluster.go

```diff
@@ -33,7 +33,7 @@ func resourceAwsEksCluster() *schema.Resource {
 		},
 
 		Timeouts: &schema.ResourceTimeout{
-			Create: schema.DefaultTimeout(15 * time.Minute),
+			Create: schema.DefaultTimeout(30 * time.Minute),
```

Comment:
This update should include a documentation update in `website/docs/r/eks_cluster.html.markdown` - thanks

Reply:
Done.